openassessment/templates/openassessmentblock/base.html

```diff
@@ -5,6 +5,17 @@
   If the new views are not enabled, `.ora-view` will be hidden, otherwise, `.ora-legacy-view`
   will be hidden.
 -->
+<!-- Hotjar Tracking Code for https://www.edx.org/ -->
+<script>
+    (function(h,o,t,j,a,r){
```

Comment:
Question, are we tracking all of the open assessment?

Reply:
Yes, hotjar integration is for all ORAs, then we have logic in views to dispatch event under certain conditions to render survey widget.

Reply:
Wouldn't that make sense to conditionally having hotjar include? If we only need it for survey, I don't see the reason to have it connect to all of the open assessment.

Reply:
Again, this is just hotjar platform integration for ORA iframe (we already have full integration in LMS). While we do want to dispatch survey event conditionally, we don't want to limit ourselves to survey only. Hotjar offers other services and we don't want to refer to code change and deployment each time we want to configure on Hotjar. Tat would be anti-pattern.

Reply:
I don't think it is anti-pattern to just do what needed to be done. If there isn't a feature require hotjar in the roadmap or near future, it seems over kill to connect hotjar to all open assessment.